### PR TITLE
README: Improve markdown formatting

### PR DESCRIPTION
**Pull request by userzimmermann**
And add `MarkdownBear` to `.coafile`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,95 @@
 # Alpakka
 
 > Licensed under the [Apache License, Version 2.0](
```
Comment (purplezimmermann) [inline README.md]: The text does not comply to the set style.

*Origin: MarkdownBear, Section: `markdown`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxppg6pdr/README.md
+++ b/tmp/tmpxppg6pdr/README.md
@@ -1,20 +1,13 @@
 # Alpakka
 
-> Licensed under the [Apache License, Version 2.0](
-http://www.apache.org/licenses/LICENSE-2.0)
+> Licensed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
 
-[![Supported Python versions](
-https://img.shields.io/pypi/pyversions/alpakka.svg)](
-https://python.org)
-[![PyPI package version](https://img.shields.io/pypi/v/alpakka.svg)](
-https://pypi.python.org/pypi/alpakka)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/alpakka.svg)](https://python.org)
+[![PyPI package version](https://img.shields.io/pypi/v/alpakka.svg)](https://pypi.python.org/pypi/alpakka)
 
-[![Travis CI build status](
-https://travis-ci.org/advaoptical/alpakka.svg?branch=master)](
-https://travis-ci.org/advaoptical/alpakka)
+[![Travis CI build status](https://travis-ci.org/advaoptical/alpakka.svg?branch=master)](https://travis-ci.org/advaoptical/alpakka)
 
-Alpakka is a python project that extends [pyang](
-https://github.com/mbj4668/pyang) to automatically generate code skeletons
+Alpakka is a python project that extends [pyang](https://github.com/mbj4668/pyang) to automatically generate code skeletons
 from YANG modules. The focus is on code stubs for the configuration of
 networks and network devices, which are controlled by NETCONF or RESCONF.
 
```

**Review by a-ts — APPROVED** [review]
Line length and code style as discussed.

**Comment by purplezimmermann** [comment]
ack https://github.com/advaoptical/alpakka/pull/3/commits/7c824551c8a5b30e2d71d9c665380eec8066aee0

**Comment by a-ts** [comment]
@purpleZ ff

**Comment by purplezimmermann** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Pull request merged**